RELEASE.md

```diff
@@ -1,19 +1,22 @@
 ## Release Finalization
 
 To cut a release:
-1. Ensure the relevant spec APIs have been released
+1. Ensure the relevant spec APIs have been released.
 1. Ensure any [migration guides](https://github.com/buildpacks/docs/tree/main/content/docs/reference/spec/migration) have been created.
-1. If applicable, ensure the README is updated with the latest supported apis (example PR: https://github.com/buildpacks/lifecycle/pull/550)
+1. If applicable, ensure the README is updated with the latest supported apis (example PR: https://github.com/buildpacks/lifecycle/pull/550).
 1. Create a release branch in the format `release/0.99.0`. New commits to this branch will trigger the `build` workflow and produce a lifecycle image: `buildpacksio/lifecycle:<commit sha>`.
 1. When ready to cut the release, manually trigger the `draft-release` workflow: Actions -> draft-release -> Run workflow -> Use workflow from branch: `release/0.99.0`. This will create a draft release on GitHub using the artifacts from the `build` workflow run for the latest commit on the release branch.
-1. Edit the release notes as necessary
-1. Perform any manual validation of the artifacts
+1. Edit the release notes as necessary.
+1. Perform any manual validation of the artifacts.
 Inside the `pack` repo:
-```
-LIFECYCLE_PATH="/path/to/created/lifecycle-v0.99.0+linux.x86-64.tgz" make acceptance
-
-$env:LIFECYCLE_PATH="C:\path\to\created\lifecycle-v0.99.0+windows.x86-64.tgz" 
-make acceptance
-```
+    On Linux:
+    ```
+    LIFECYCLE_PATH="/path/to/created/lifecycle-v0.99.0+linux.x86-64.tgz" make acceptance
```

Comment:
Are these relevant now? Can we pass a specific lifecycle via a `pack` arg now?

Reply:
`--lifecycle-image` is a thing now, but not `--lifecycle`. But come to think of it, I should update the description to include the use of `LIFECYCLE_IMAGE="buildpacksio/lifecycle:<random-tag>"`.

Reply:
Done!